internal/parse/parser_test.go

```diff
@@ -191,28 +195,34 @@ func TestRound(t *testing.T) {
 	}
 }
 
-// We return a proper error when we find an unbound string literal
 func TestUnfinishedStringLiteral(t *testing.T) {
-	sql := "SELECT foo FROM t WHERE x = 'dddd"
-	parser := parse.NewParser()
-	_, err := parser.Parse(sql)
-	assert.Equal(t, fmt.Errorf("cannot parse expression: missing right quote in string literal"), err)
-}
-
-func TestUnfinishedStringLiteralV2(t *testing.T) {
-	sql := "SELECT foo FROM t WHERE x = \"dddd"
-	parser := parse.NewParser()
-	_, err := parser.Parse(sql)
-	assert.Equal(t, fmt.Errorf("cannot parse expression: missing right quote in string literal"), err)
-}
+	var testTable = []struct {
+		name string
+		sql  string
+		err  string
+	}{
+		{
```

Comment:
The two parenthesis above can be merged together. That's below as well on the ending (line 217).

This is the same style used right above in this same diff.

Reply:
Changed.